extensions/apps/extensions/src/components/pages/extension-release-manager/edit-test-release-page.tsx

```diff
@@ -115,7 +115,7 @@ export const EditTestReleasePage: React.FC<EditTestReleasePageProps> = ({
         ) {
           timeout = setTimeout(() => {
             navigateTo({
-              appName: baseAppInfo.name,
+              appName: baseAppInfo?.name,
```

Comment:
this is already handled in my pr, we'll have conflicts if left like this

Reply:
ok, I reverted the change